modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/AdminInputs/index.js

```diff
@@ -9,37 +9,13 @@
  * distribution rights of the Software.
  */
 
-import {useLazyQuery} from '@apollo/client';
 import ClayForm, {ClayInput} from '@clayui/form';
-import {useEffect, useState} from 'react';
 import {Input} from '../../../../components';
-import useDebounce from '../../../../hooks/useDebounce';
-import {getBannedEmailDomains} from '../../../../services/liferay/graphql/queries';
+import useBannedDomains from '../../../../hooks/useBannedDomains';
 import {isValidEmail} from '../../../../utils/validations.form';
 
 const AdminInputs = ({admin, id}) => {
-	const debouncedEmail = useDebounce(admin?.email, 500);
-	const [bannedDomain, setBannedDomain] = useState(debouncedEmail);
-
-	const [fetchBannedDomain, {data}] = useLazyQuery(getBannedEmailDomains);
-	const bannedDomainsItems = data?.c?.bannedEmailDomains?.items;
-
-	useEffect(() => {
-		const emailDomain = debouncedEmail.split('@')[1];
-
-		if (emailDomain) {
-			fetchBannedDomain({
-				variables: {
-					filter: `domain eq '${emailDomain}'`,
-				},
-			});
-
-			if (bannedDomainsItems?.length) {
-				setBannedDomain(bannedDomainsItems[0].domain);
-			}
-		}
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [bannedDomainsItems, debouncedEmail]);
+	const bannedDomains = useBannedDomains(admin?.email, 500);
```

Comment:
I think it will be better if declare a constant for this value (500) , what do you think?

Reply:
Nice, done!